cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js

```diff
@@ -0,0 +1,44 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearchAndFilter from '../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix, { randomFourDigitNumber } from '../../support/utils/stringTools';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+const item = {
+  instanceName: `inventory_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+const wrongMessage = 'PO number does not exist';
+const wrongPO = `${randomFourDigitNumber()}`;
+
+describe('Inventory interaction', () => {
+  before('Create test data', () => {
+    cy.getAdminToken();
+    InventoryInstances.createInstanceViaApi(item.instanceName, item.itemBarcode);
```

Comment:
Need to create user with permissions
Existing user with follow permissions is logged in:
-Inventory: Create order from instance
-Inventory: View, create, edit holdings
-Inventory: View, create, edit instances
-Inventory: View, create, edit items
-Orders: Can view Orders and Order lines